tools/lib/provision_inner.py

```diff
@@ -33,7 +33,7 @@
 from tools.setup.generate_zulip_bots_static_files import generate_zulip_bots_static_files
 from version import PROVISION_VERSION
 
-VENV_PATH = "/srv/zulip-py3-venv"
+VENV_PATH = os.path.join(ZULIP_PATH, ".venv")
```

Comment:
TODO: remove the old `source /srv/zulip-py3-venv/bin/activate` line from the user’s `.profile`.